lexicons/com/atproto/identity/sendPlcOp.json

```diff
@@ -0,0 +1,20 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.identity.sendPlcOp",
```

Comment:
maybe `submitPlcOp`? that is the verb used in the description and feels a bit better, but not a strong feeling.

Reply:
yeah I like that 👍